x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/processes/index.tsx

```diff
@@ -29,7 +31,11 @@ import { SummaryTable } from './summary_table';
 import { ProcessesTable } from './processes_table';
 import { parseSearchString } from './parse_search_string';
 
-const TabComponent = ({ currentTime, node, nodeType }: TabProps) => {
+interface ProcessesProps extends Omit<TabProps, 'node' | 'onClose' | 'options'> {
+  node: InfraWaffleMapNode | HostNodeRow;
+}
+
+const TabComponent = ({ currentTime, node, nodeType }: ProcessesProps) => {
```

Comment:
In the future, we might want to start moving components that are now used in both Hosts View and Inventory UI to `public/components/inventory` as they are now common components. Ideally, we shouldn't create interdependencies between pages.

Reply:
That's a really good suggestion, thanks! I would add a separate issue for that as it already has some imports from metrics explorer and infra inside so it will be a bigger refactoring.